### ISSUE TITLE
Automate installer for unattended setup

### ISSUE THREAD

**Issue by georgerb** [remoteit/installer#9]
Is there a way to script the installer itself? Ex:
```connectd_installer -u user@email.com -p somepasswordtoken -n devicename -p someport ```

**Comment by Digital-Larry** [comment]
Hi George, please try the attached file.  Hopefully it is self explanatory.  Let me know if not.

[connectd_register.zip](https://github.com/remoteit/installer/files/3073920/connectd_register.zip)

**Comment by funwhilelost** [comment]
@Digital-Larry I'm not able to get this script working.  It says "You have entered either an incorrect username or password." but I've verified the credentials are correct.  Is it possible the API key is wrong?  I noticed in the remote.it account page there are both a "Developer API Key" and "Access Keys".

There is also a param in the script called `AUTH_HASH` which isn't explained in the comments.
